appveyor.yml

```diff
@@ -0,0 +1,22 @@
+build: false
+shallow_clone: false
+platform: 'x86'
+clone_folder: c:\projects\symfony
+
+init:
+  - SET PATH=c:\tools\php\;%PATH%
+
+install:
+  - cinst gow
+  - cinst php
+  - cd c:\tools\php
+  - php -i
+  - ps: Start-FileDownload 'https://getcomposer.org/composer.phar'
+  - echo @php "%~dp0composer.phar" %*>composer.bat
+  - cd c:\projects\symfony
+  - bash phpunit install
+  - bash -c 'if [ "$APPVEYOR_REPO_BRANCH" = "master" ]; then export COMPOSER_ROOT_VERSION=dev-master; else export COMPOSER_ROOT_VERSION="$APPVEYOR_REPO_BRANCH".x-dev; fi; composer install --prefer-dist --no-progress'
+
+test_script:
+  - cd c:\projects\symfony
+  - gfind src/Symfony -mindepth 3 -type f -name phpunit.xml.dist -printf '%h\n' | xargs -l 'bash phpunit --exclude-group tty,benchmark,intl-data'
```

Comment:
do we need to exclude the tty group ?

We exclude it on Travis from the parallel run because it fails in such case (parallel does not run phpunit from a tty)but we run them separately after that.